include/fast_float/parse_number.h

```diff
@@ -99,7 +99,24 @@ fastfloat_really_inline bool rounds_to_nearest() noexcept {
   //
   // Note: This may fail to be accurate if fast-math has been
   // enabled, as rounding conventions may not apply.
+  #ifdef _MSC_VER
+  #   pragma warning(push)
+  //  todo: is there a VS warning?
```

Comment:
I recommend linking to https://stackoverflow.com/questions/46079446/is-there-a-warning-for-floating-point-equality-checking-in-visual-studio-2013

Reply:
indeed, I found that one as well but since there was nothing I left it at that. But I'll follow your recommendation.